dataplane/saiserver/hostif.go

```diff
@@ -155,13 +155,21 @@ func (hostif *hostif) CreateHostif(ctx context.Context, req *saipb.CreateHostifR
 				return nil, err
 			}
 		}
+
+		cpuPortReq := &saipb.GetSwitchAttributeRequest{Oid: 1, AttrType: []saipb.SwitchAttr{saipb.SwitchAttr_SWITCH_ATTR_CPU_PORT}}
```

Comment:
What does OID 1 mean? Is it reserved for CPU port?

Reply:
made a const, OID 1 is always the switch id. here we are queried the switch attribute to figure out the ID of the CPU port